bentoml/sklearn.py

```diff
@@ -36,14 +36,10 @@ def _get_model_info(
 ) -> t.Tuple["Model", PathType]:
     model = model_store.get(tag)
     if model.info.module != __name__:
-        if model.info.module == "bentoml.mlflow":
-            pass
-        else:
-            raise BentoMLException(  # pragma: no cover
-                f"Model {tag} was saved with module"
-                f" {model.info.module}, failed loading"
-                f" with {__name__}."
-            )
+        raise BentoMLException(  # pragma: no cover
```

Comment:
cc @aarnphm - this check should be removed right?

Reply:
we can just have it so that users can't load untrusted pickle file, since for sklearn we don't even use sklearn at all

Reply:
We still have the module check, I mean the removed part that allows loading bentoml.mlflow saved model with bentoml.sklearn module

Reply:
oh it should be removed in some of the other PR i had